src/clr-icons/clr-icons.spec.ts

```diff
@@ -35,18 +35,23 @@ import { clrIconSVG } from './utils/svg-tag-generator';
 // This is a base test object for all sets
 /* tslint:disable:no-string-literal */
 const ALL_SETS = [
-  { name: 'Commerce shapes', shapes: CommerceShapes, randomShape: { 'e-check': CommerceShapes['e-check'] } },
-  { name: 'Essential shapes', shapes: EssentialShapes, randomShape: { pencil: EssentialShapes['pencil'] } },
-  { name: 'Social shapes', shapes: SocialShapes, randomShape: { star: SocialShapes['star'] } },
-  { name: 'Media shapes', shapes: MediaShapes, randomShape: { play: MediaShapes['play'] } },
-  { name: 'Travel shapes', shapes: TravelShapes, randomShape: { car: TravelShapes['car'] } },
-  { name: 'Technology shapes', shapes: TechnologyShapes, randomShape: { cpu: TechnologyShapes['cpu'] } },
-  { name: 'Chart shapes', shapes: ChartShapes, randomShape: { 'bar-chart': ChartShapes['bar-chart'] } },
-  { name: 'Text Edit shapes', shapes: TextEditShapes, randomShape: { bold: TextEditShapes['bold'] } },
+  { name: 'Core shapes', shapes: CoreShapes, randomShape: ClrShapeCheck },
+  { name: 'Commerce shapes', shapes: CommerceShapes, randomShape: ClrShapeECheck },
+  { name: 'Essential shapes', shapes: EssentialShapes, randomShape: ClrShapePencil },
+  { name: 'Social shapes', shapes: SocialShapes, randomShape: ClrShapeStar },
+  { name: 'Media shapes', shapes: MediaShapes, randomShape: ClrShapePlay },
+  { name: 'Travel shapes', shapes: TravelShapes, randomShape: ClrShapeCar },
+  { name: 'Technology shapes', shapes: TechnologyShapes, randomShape: ClrShapeCPU },
+  { name: 'Chart shapes', shapes: ChartShapes, randomShape: ClrShapeBarChart },
+  { name: 'Text Edit shapes', shapes: TextEditShapes, randomShape: ClrShapeBold },
 ];
 /* tslint:enable:no-string-literal */
 
```

Comment:
Can we add a test case scenario where we don't run `init()` and to see that it doesn't set the global object?